### PR TITLE
Feature/#648 table styles scope

### PR DESCRIPTION
**Pull request by NicolaSaunders**
Proposed solution for https://github.com/w3c/w3c-website/issues/648

### CODE REVIEW

**Review by deniak — APPROVED** [review]

File: assets-src/styles/sass/30-base/_tables.scss [hunk 1]
```diff
@@ -34,12 +34,9 @@ table caption {
 }
 
 th {
+    background-color: $deep-blue;
```
Comment (deniak) [inline assets-src/styles/sass/30-base/_tables.scss:37]: Discussing with @vivienlacourba, only the `<th>`s in a `<thead>` or `<tfoot>` should have that blue background.

**Review by deniak — CHANGES_REQUESTED** [review]

File: assets-src/styles/sass/30-base/_tables.scss (hunk 1 above)
Comment (NicolaSaunders) [reply assets-src/styles/sass/30-base/_tables.scss:42]: Have pushed a further changed for those requirements.

**Review by deniak — APPROVED** [review]

**Comment by deniak** [comment]
@NicolaSaunders thanks for that PR.
This is unrelated but what do you think of removing the compiled assets to this repository. The [actions we have in place](https://github.com/w3c/w3c-website-templates-bundle/tree/main/.github/workflows) will compile them before uploading.

**Comment by NicolaSaunders** [comment]
@deniak No problem!

Happy to discuss how to handle compiled assets - let's move that conversation out of this PR though.

**Pull request merged**